lcli/src/main.rs

```diff
@@ -92,6 +92,27 @@ fn main() {
                         .help("Path to output a SSZ file."),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("pretty-ssz")
+                .about("Parses a file of raw (not hex-encoded) SSZ bytes")
+                .arg(
+                    Arg::with_name("type")
+                        .index(1)
+                        .value_name("TYPE")
+                        .takes_value(true)
+                        .required(true)
+                        .possible_values(&["SignedBeaconBlock"])
+                        .help("The schema of the supplied SSZ."),
+                )
+                .arg(
+                    Arg::with_name("path")
+                        .index(2)
+                        .value_name("HEX")
```

Comment:
[drive by review]

This parameter should be called `FILE` or something other than `HEX`

Reply:
Addressed :)